datasets/ar/repet/ar_repet.yml

```
@@ -123,3 +123,16 @@ lookups:
           - Liberation of al-Sham Commission
           - Liberation of the Levant Organisation Tahrir al-Sham
           - Tahrir al-Sham Hay’at
+  type.identifier:
+    lowercase: true
+    normalize: true
+    options:
+      - match: 
+          - Unidad de Información FinancieraArE.00005
+          - HArE.001 y HArE.002
+        value: null
```

```suggestion
      - match: Unidad de Información FinancieraArE.00005
        value: ArE.00005
      - match: HArE.001 y HArE.002
        values:
          - HArE.001
          - HArE.002
```

let's not lose these.

It looks like they're using some UN SC style ID formats that aren't documented on the consolidated list.

```jq .properties.unscId data/datasets/ar_repet/entities.ftm.json |sort|uniq -c
   1   "HArE.00011"
   1   "HArE.001 y  HArE.002"
   1   "HArE.001"
   1   "HArE.002"
   1   "HArP.00009"
   1   "HArP.00010"
   1   "HArP.00011"
   1   "HArP.00012"
   1   "HArP.00013"
   1   "HArP.00014"
   1   "HArP.00015"
   1   "HArP.00016"
   1   "HArP.00017"
   1   "HArP.001"
   1   "HArP.002"
   1   "HArP.003"
   1   "HArP.005"
   1   "HArP.006"
   1   "HArP.007"
   1   "OArE.00012"
   1   "QDe.001"
   1   "QDe.002"
   1   "QDe.003"
```

if you wanna be fancy, you can replace `ArE.` with ` y ArE.` before spliting on ` y ` then values like 
```
1   "QDe.156"
   1   "QDe.157"
   1   "QDe.158"
   1   "QDe.159ArE.00006"
   1   "QDe.160"
   1   "QDe.161"
   1   "QDe.163"
```
will be happier